modin/utils.py

```diff
@@ -32,7 +32,7 @@
 import pandas
 import numpy as np
 
-from pandas.util._decorators import Appender
+from pandas.util._decorators import Appender  # type: ignore
```

Comment:
If I remove this, mypy fails. I don't understand how my changes brought about that situation, but here we are.